css/camera_controls.css

```diff
@@ -9,12 +9,12 @@
 
 .cc-container .btn {
     border: none;
-    color: #d8d8d8;
-    background-color: #25222e;
+    color: var(--white-one);
+    background-color: var(--dark-one);
 }
 
 .cc-container .btn:hover {
     border: none;
-    color: #d8d8d8;
-    background-color: #4A4658;
+    color: var(--white-one);
+    background-color: var(--gray-one);
```

Comment:
oh wait! this says `gray-one`! is this supposed to be `gray-two`?

Reply:
These variable names are in alignment with color sheet now, but `color` wasn't actually being applied because antd was overriding it, so I just added an `!important` tag...

Reply:
@meganrm mentioned that we should avoid `!important`

Antd makes it hard to target these rules, and the simplest way was to add ids to the buttons, which I did here.

We can do like the website repo and set up an `ant-vars.less` file to override styling globally but that seems out of scope and I would write a new issue for that and do a separate PR down the line. Also it seems antd will not be supporting `less` in version 5 which is what we currently have installed, so we should iron out a dependency plan...